packages/cdc-react/src/components/Card/Card.tsx

```diff
@@ -0,0 +1,50 @@
+import { Button } from "../Button/Button";
+import "./Card.scss";
+
+interface CardProps {
+  cardLayout: "horizontal" | "vertical-left" | "vertical-right";
+  showTitleTop?: boolean; // only works for horizontal layout
+  cardSectionText?: string;
+  cardAction: string; // cannot be optional according to the design system
```

Comment:
I think there should be an onAction mouse event handler.

Reply:
For the button or for the card itself?

Reply:
I think just the button.

Reply:
I think we could add a prop of `cardActionButtonHandler` to account for this.

Reply:
@cfarmer-fearless, thoughts? I don't think this is necessary @ian-sawyer. We want the user of the component to decide how to handle the action.

Reply:
I agree with both of you.  We need a prop to allow the user to define a handler for the action click.